doc/plotting.rst

```diff
@@ -174,14 +174,42 @@ entire figure (as for matplotlib's ``figsize`` argument).
 
 .. _not equivalent to matplotlib's: https://github.com/mwaskom/seaborn/issues/746
 
+
+Multiple lines showing variation along a dimension
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to make line plots of two-dimensional data by calling ``xarray.plot.line()`` with appropriate arguments. Consider the 3D variable ``air``
+
+.. ipython:: python
+
+    air
+
+We can use line plots to check the variation of air temperature at three different latitudes along a longitude line:
+
+.. ipython:: python
+
+    @savefig plotting_example_multiple_lines_x_kwarg.png
+    air.isel(lon=10, lat=[19,21,22]).plot.line(x='time')
+
+It is required to explicitly specify either
+
+1. ``x``: the dimension to be used for the x-axis, or
+2. ``hue``: the dimension you want to represent by multiple lines.
+
+Thus, we can make the previous plot by specifying ``hue='lat'`` instead of ``x='time'``. If required, the automatic legend can be turned off using ``add_legend=False``.
+
+.. ipython:: python
+
+    @savefig plotting_example_multiple_lines_hue_kwarg.png
```

Comment:
I would omit this example. `add_legend=False` doesn't need an example to make it clear what it does, and plots take up lots of space in the docs.

Reply:
It also illustrates hue, but we could just leave that as a text description.

Is there eat you would prefer?

Reply:
I'm OK with it in the text description.